.github/workflows/build.yml

```
@@ -169,6 +169,8 @@ jobs:
           REDIS_PORT=${{ secrets.REDIS_PORT }}
           REDIS_PASSWORD=${{ secrets.REDIS_PASSWORD }}
           
+          ELASTICSEARCH_HOST=${{ secrets.ELASTICSEARCH_HOST }}
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)